torch/_torch_docs.py

```diff
@@ -1242,6 +1242,46 @@ def merge_dicts(*dicts):
     tensor([ True, False, False])
 """.format(**common_args))
 
+add_docstr(torch.bitwise_left_shift,
+           r"""
+bitwise_left_shift(input, other) -> Tensor
+
+Computes the bitwise shift of :attr:`input` to the left by :attr:`other` bits.
+The input tensor must be of integral types.
```

Comment:
This is interesting because the OpInfo shows that floating point types works, and testing in PyTorch also shows that left and right shift work with floats. What about adding a check that both input and other are integer types in the code? Then this sentence would be updated to: 

"Both :attr:\`input\` and :attr:\`other\` must have integer dtypes."

It can be its own paragraph after the first sentence.

Reply:
Another important here, however, is that this function does not participate in type promotion. The returned dtype should be the same as input's dtype. So I guess this could say:

"Both :attr:`input` and :attr:`other` must have integer dtypes. The result will have the same dtype as :attr:\`input\`."